### DISCUSSION TITLE
Nginx Loadbalance Stuck on Destroy

### DISCUSSION THREAD

**Discussion by donydonald1** [kube-hetzner/terraform-hcloud-kube-hetzner#580] [Q&A]
Not sure if there is a fix for this but but thought i voice it out... TF is unable to delete ingress loadbalancer for some reasons including autoscalar vms. The fix is to manually delete them.

**Comment by valkenburg-prevue-ch** [comment]
For this reason I have those helm charts which create loadbalancers as part of the same terraform project as the kube.tf story. So that a `terraform destroy` also removes that loadbalancer. But I would love to hear about a better practice.

**Comment by mysticaltech** [comment]
@donydonald1 @valkenburg-prevue-ch As you might know already, this happens because it's not Terraform that creates the ingress LB but the Hetzner CCM on request of the Ingress controller. 

Currently working on a solution to remove the need to manually delete the LB.
